src/components/MovieCard.tsx

```diff
@@ -1,10 +1,18 @@
 import React from "react";
-import PropTypes from "prop-types";
-import { useDispatch } from "react-redux";
 import Shorten from "./Shorten";
 
 import { removeMovieActionCreator } from "../store/actionCreators";
 import { Link } from "react-router-dom";
+import { useAppDispatch } from "../store/selectors";
+
+interface MovieInterface {
```

Comment:
since you'd need this in your state anyway, I'd either make separate file for it here or move this type in store types and export from here. You can also create separate folder in root called `types` where you'll store types like this (that you'd need to reuse in multiple places). Just make sure you go with chosen approach for all files it needed to